src/routes/Onboarding/ProjectMenu.tsx

```diff
@@ -51,7 +51,7 @@ export default function ProjectMenu() {
                 </a>{' '}
                 for that. We aren't hosting files for you at this time but are
                 considering supporting it in the future, so we're building
-                Modeling App with a browser-first experience in mind.
+                Design Studio with a browser-first experience in mind.
```

Comment:
Is the formal name "Design Studio" or "Zoo Design Studio". The below in `src/routes/Onboarding/Sketching.tsx` has the latter.

Reply:
Within the context of the app I think "Design Studio" is fine. If it's ever ambiguous or if it's the first time in the audience's mind that the app is being learned about (like a blog post, where we don't know their familiarity), I'd use "Zoo Design Studio".